projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/initVDKCell.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { INotebookTracker } from '@jupyterlab/notebook';
+
+/**
+ * This func adds a cell with the necessary commands to enable VDK for a Jupyter notebook
+ */
+export function initVDKCell(notebookTracker: INotebookTracker) {
```

Comment:
Here can you rename it to be more specific since here we initialise a specific cell for ipython conf
I will be adding another function that initialises VDK cells on tranform operation